.github/workflows/ios_build_hotfix_release.yml

```diff
@@ -130,3 +137,16 @@ jobs:
       commit-sha: ${{ needs.update_asana.outputs.commit_sha }}
       release-type: internal # Pre-release for now, after submitting to app store we will run ios-tag-release-update-asana again manually with 'hotfix' release type to finish up
     secrets: inherit
+
+  report_failure:
+    name: Report Failure
+    needs: [ assert_release_branch, run_tests, update_asana, prepare_release, tag_and_merge ]
+    if: always() && (needs.run_tests.result == 'failure' || needs.update_asana.result == 'failure' || needs.prepare_release.result == 'failure' || needs.tag_and_merge.result == 'failure')
```

Comment:
Here we “need” all other jobs, and run whenever any of them, except the first one, fails. If the first one fails, we don’t know the task ID to comment on.